standard/links/commons/links.lua

```diff
@@ -349,6 +349,112 @@ local ICON_KEYS_TO_RENAME = {
 	douyin_page = 'douyin',
 }
 
+local MATCH_ICONS = {
+	dbstats = {icon = 'File:Diabotical icon.png', text = 'QuakeLife matchpage'},
+	qrindr = {icon = 'File:Quake Champions icon.png', text = 'Qrindr matchpage'},
+	pf = {icon = 'File:Plus Forward icon.png', text = 'Plus Forward matchpage'},
+	interview = {icon = 'File:Int_Icon.png', text = 'Interview'}, -- OR Interview32.png?
+	cdl = {
+		icon = 'File:Call of Duty League lightmode.png',
+		iconDark = 'File:Call of Duty League darkmode.png',
+		text = 'Call of Duty League matchpage'
+	},
+	breakingpoint = {
+		icon = 'File:Breaking Point GG icon lightmode.png',
+		iconDark = 'File:Breaking Point GG icon darkmode.png',
+		text = 'Breaking Point matchpage'
+	},
+	royaleapi = {icon = 'File:RoyaleAPI_allmode.png', text = 'RoyaleAPI Match Page'},
+	vod = {
+		icon = 'File:VOD Icon.png',
+		text = 'Watch VOD'
+	},
```

Comment:
this is not needed, right?
like its display gets build via VodLink instead

mind due to vod being in the processing data but also having its own lpdb field we might get duplicate data for vods (once in match_vod/match2game_vod and once in links)